styles/style.css

```diff
@@ -0,0 +1,620 @@
+  background-size: 17%, 21%;
+  background-position: 0 22%, 100% -0.5%;
+}
+
+.about-text {
+  color: #42526e;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  margin-bottom: 24px;
+}
+
+.about-btn {
+  margin-bottom: 59px;
+}
+
+.cv-container {
+  display: grid;
+  grid-template-rows: repeat(3, 352px);
+  grid-template-columns: 1fr;
+  grid-row-gap: 24px;
+}
+
+.cv-cell {
+  height: 352px;
+  border: 1px solid #c1c7d0;
+  border-radius: 8px;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  background-color: #ebf0ee;
+  margin: 0 12px;
+  padding-top: 40px;
+}
+
+.cv-img {
+  justify-content: center;
+  align-items: center;
+  width: 80px;
+  height: 80px;
+  background-color: #fff;
+  border-radius: 50%;
+  text-align: center;
+  text-decoration: none;
+  line-height: 80px;
+}
+
+.cv-cell-title {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  color: #344563;
+  text-align: center;
+  align-self: stretch;
+  margin: 64px 0 24px 0;
+}
+
+.cv-feat-cont {
+  justify-content: space-around;
+  align-items: center;
+  flex-wrap: wrap;
+  align-self: stretch;
+}
+
+.cv-feat-btn {
+  display: inline-block;
+  box-sizing: border-box;
+  padding: 12px;
+  margin-bottom: 8px;
+  background-color: #fff;
+  border-radius: 8px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #36b37e;
+  text-decoration: none;
+}
+
+.cv-feat-cont-margin-2 {
+  margin: 0 30px;
+}
+
+.form-container {
+  width: 100%;
+  flex-direction: column;
+  align-items: center;
+  background-image: url('../images/form.png');
+  background-repeat: no-repeat;
+  background-position: 100% 67%;
+}
+
+.form-text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  text-align: center;
+  padding: 0 6% 58px 6%;
+}
+
+.form {
+  width: 87%;
+  flex-direction: column;
+  align-items: center;
+}
+
+.box {
+  width: 100%;
+  margin-bottom: 24px;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+  box-sizing: border-box;
+  padding: 15px 16px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.box-textarea {
+  height: 114px;
+  color: #091e42;
+}
+
+.form-btn {
+  border: none;
+  cursor: pointer;
+}
+
+.social-media-form {
+  margin-top: 44px;
+  margin-bottom: 19px;
+}
+
+.email {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  color: #42526e;
+  margin-bottom: 290px;
+}
+
+.final-line {
+  border: 2px solid #000;
+  border-radius: 20px;
+  width: 35%;
+  margin-bottom: 8px;
+}
+
+@media screen and (min-width: 376px) {
+  body {
```

Comment:
- [ ] Please, consider that the value for `min-width` property should be in range of `768px` or wider 👍